src/main/io/serial_cli.c

```diff
@@ -547,6 +547,10 @@ static const char * const lookupTablePwmProtocol[] = {
 #endif
 };
 
+static const char * const lookupTablePwmPolarity[] = {
+    "POSITIVE", "NEGATIVE"
+};
```

Comment:
Would `motor_pwm_idle : LOW, HIGH` be better or you suggest a boolean like @mikeller?